docs/_docs/examples/recipe.md

```diff
@@ -309,3 +330,76 @@ cleaned_files = replace_identifiers(dicom_files=dicom_files,
                                     output_folder='/home/vanessa/Desktop',
                                     overwrite=True)
 ```
+
+## Groups
+
+More advanced usage of header actions would be to define a group of values (the content of the 
+header fields) or field names (the names themselves) to use in an action. This corresponds
+to `%fields` (a list of fields) and `%values` (a list of values from fields) to parse
+at the onset of the dicom load, and use later in a recipe. Here is how that might look
+in a recipe:
+
+```
+FORMAT dicom
+
+%values cookie_names
+SPLIT PatientID by=" ";minlength=4
+
+%values operator_names
+FIELD startswith:Operator
+
+%fields instance_fields
+FIELD contains:Instance
+
+%header
+
+ADD PatientIdentityRemoved Yes
+REPLACE values:cookie_names var:id
+REPLACE values:operator_names var:source_id
+REMOVE fields:instance_fields
+```
+
+In the above, we define two lists of values (operator_names and cookie_names)
+and a list of fields (instance_fields). The sections read as follows:
+
+ - create a list of values called `cookie_names` that are from the PatientID field that is split by a space with a minimum length of 3
```

Comment:
Very minor comment... This statement doesn't match the code sample above.  In the code sample, minlength=4.

Reply:
Fixed! I remember writing this, changing my mind about the value to use, and then clearly only doing the change in one place. Thanks for catching this!